src/reducers/settings.js

```diff
@@ -5,9 +5,10 @@ import type { SettingsState } from "../types";
 const INITIAL_STATE: SettingsState = {
   maxPerPage: 200,
   singleServer: null,
+  authMethods: ["basicauth", "fxa", "ldap"]
 };
 
 // This is basically a noop as we don't support settings update at runtime (yet).
 export default function settings(state: SettingsState = INITIAL_STATE): SettingsState {
-  return state;
+  return {...INITIAL_STATE, ...state};
```

Comment:
Well, that does the same thing no? :) I mean, no action is ever processed here, and so state never changes...